data_steward/gcloud/gcs/__init__.py

```diff
@@ -59,37 +59,32 @@ def get_blob_metadata(self, blob: Blob) -> dict:
         }
         return metadata
 
-    def get_drc_bucket(self) -> str:
-        result = os.environ.get('DRC_BUCKET_NAME')
-        return result
+    def get_drc_bucket(self) -> Bucket:
+        return self.get_bucket(os.environ.get('DRC_BUCKET_NAME'))
 
-    def get_hpo_bucket(self, hpo_id: str) -> str:
+    def get_hpo_bucket(self, hpo_site: str) -> Bucket:
```

Comment:
This should be `hpo_id`. It is the identifier used for HPO sites in the project.

Reply:
Committed